scripts/extractLang.js

```diff
@@ -1,86 +0,0 @@
-// This file is part of Prusa-Connect-Local
```

Comment:
How the translations will be now updated?

Reply:
This extract from touch-ui, it's good to create a script for phrase.